app/templates/components/mesh-manager.hbs

```diff
@@ -1,5 +1,7 @@
 {{#if targetItemTitle}}
-  <h2>{{t 'selectMeshFor' title=targetItemTitle}}</h2>
+ <span class='specific-title'>
```

Comment:
Well <h3> was strangely even larger - I was trying to save some space.  Does it need to be reverted to <h2>?